rdkit/Chem/EState/UnitTestVSA.py

```diff
@@ -12,52 +12,44 @@
 
 """
 from __future__ import print_function
-from rdkit import RDConfig
-import unittest, os
+
+import os
+import unittest
+
 from rdkit import Chem
 from rdkit.Chem.EState import EState_VSA
-import os.path
 
 
-def feq(n1, n2, tol=1e-4):
-  return abs(n1 - n2) <= tol
+class TestCase(unittest.TestCase):
 
+  @staticmethod
```

Comment:
When I looked at this test it took me a while to figure out what it was doing. Understanding required knowing a few things about how the rest of the code works. Why do we get the expected values from the molecule, how do we access the functions?

By separating the loading of the validation data (that could be a better name) from the comparison it seemed to me clearer. Looking at it again, i think the population of the property function dictionary should be moved to the test for a clearer separation.